### DISCUSSION TITLE
4-Node QBFT Network Cannot Reach Consensus When There is 1 Node Failure

### DISCUSSION THREAD

**Discussion by peter-chung-xfers** [ConsenSys/quorum#1398] [Q&A]
#### Expected Behavior 

a network with 3f+1 nodes should be able to tolerate 1 node failure

#### Actual Behavior 

the network is frozen by 1 node failure with message `Looking for peers peercount=2 tried=0 static=4`

logs: 
![Screenshot from 2022-05-10 14-55-58](https://user-images.githubusercontent.com/81944717/167567077-bea2e2c3-d738-40fb-9149-67d0cf151c4a.png)

**Comment by baptiste-b-pegasys** [comment]
Hello, peercount=2 suggests that you have only 2 peers in your network out of 4 (static). The qualified majority is required, so 3 peers has to be connected.

**Reply by peter-chung-xfers** [reply]
Is the local node itself considered as a peer too?
This is the peer count from a healthy network which is 3
![Screenshot from 2022-05-10 16-40-28](https://user-images.githubusercontent.com/81944717/167586981-175a1726-12c8-4c73-befa-5140e8bdb6cd.png)
